### PR TITLE
The global ID for all elements is now stored in the primal grid struc…

### PR DESCRIPTION
**Pull request by economon**
…ture (same idea/methods as the global ID at each vertex). This is useful for finite element calculations in particular (FEA).

We will have to fast track this one, as it is needed for a paper. However, I wanted everybody to see the addition and have a chance to comment on it.

We'll plan to get the output and incompressible PRs in next.

### CODE REVIEW

File: SU2_CFD/src/driver_structure.cpp [hunk 1]
```diff
@@ -612,6 +612,15 @@ void CDriver::Geometrical_Preprocessing() {
       cout << "Setting the multigrid structure." << endl;
 
   }
+  
+#ifdef HAVE_MPI
+  MPI_Barrier(MPI_COMM_WORLD);
+#endif
+  for (unsigned long iElem = 0; iElem < geometry_container[ZONE_0][MESH_0]->GetnElem(); iElem++)
```
Comment (rsanfer) [inline SU2_CFD/src/driver_structure.cpp]: Just as a quick comment to remember to remove this before merging with develop.

**Comment by economon** [comment]
Yikes! Thanks for the comment.. thought I got that out of there.

Uploading the fix, and if everything checks out ok, we'll merge this in right away.

**Comment by economon** [comment]
I think we're ready. If you give it a thumbs up after checking that it's working for you, @rsanfer, I'll merge it in.

**Comment by rsanfer** [comment]
I added the value of the global (real) number of elements before the partitioning. I needed this for GlobalToLocal vectors and load of information from file. @economon, can you double check it's all fine with this?

**Comment by economon** [comment]
LGTM. Just updated with develop and will merge in if the tests pass.

**Pull request merged**